docker/build_clio.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -ex
+
+BUILD_DIR=build
+BUILD_CONFIG=Release
+TESTS=False
+NPROC=$(($(nproc) - 2))
+
+source /opt/rh/devtoolset-11/enable
+source /opt/rh/rh-python38/enable
+
+pip install "conan<2"
+conan remote add --insert 0 conan-non-prod http://18.143.149.228:8081/artifactory/api/conan/conan-non-prod || true
+# TODO: adjust --build parameters when remote is updated with binaries built from
+conan install . \
+    --build b2  \
+    --build missing \
+    --output-folder ${BUILD_DIR} \
+    --options tests=${TESTS} \
+    --options "xrpl/*:rocksdb=False" \
```

Comment:
Do you need it here if you added it to the conanfile?

Reply:
Nope, it doesn't need to be here.